ooyala_player/transcript.py

```diff
@@ -186,10 +188,10 @@ def get_transcript_by_threeplay_id(api_key, transcript_id, threeplay_id):
 
         return transcript_content
 
-    def render(self):
+    def render(self, i18n_service=None):
         """
         Render in interactive transcript
         """
-        return render_template('templates/html/ooyala_transcript.html', {
+        return loader.render_django_template('templates/html/ooyala_transcript.html', {
```

Comment:
Can you format this code to follow the edX python guidelines?
```
return loader.render_django_template(
    'templates/html/ooyala_transcript.html', 
    {
        'self': self
    }, 
    i18n_service=i18n_service
)
```

Reply:
what guildelines you are talking about? Can you share the link of it?

Reply:
These: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html :grin:

Reply:
see what I have here.
https://imgur.com/a/XkZnvlX